src/app/productManagement/relatedProducts/templates/relatedProducts.html

```diff
@@ -0,0 +1,69 @@
+<div class="row">
+    <div class="col-sm-12">
+        <h4>Related Products  <span class="badge">{{relatedProduct.relatedProducts.Meta.TotalCount || 0}}</span></h4>
+        <div class="row">
+            <div class="col-sm-8 col-md-9">
+                <ui-select multiple ng-model="relatedProduct.selectedProducts" close-on-select="false">
+                    <ui-select-match placeholder="Search and select Products to add...">
+                        {{$item.Name}}
+                        <span ng-bind="$select.selected.Name"></span>
+                    </ui-select-match>
+                    <ui-select-choices repeat="product in (relatedProduct.uiSelectProducts.Items | filter: $select.search) track by $index" refresh="relatedProduct.listAllProducts($select.search)" refresh-delay="300">
```

Comment:
Doesn't seem like a good idea to list all products just so they can select from ui-select. What's going to happen if someone like DFSI was going to use this?

I'm pretty sure ui-select offers an asynchronous way of retrieving stuff so that you don't have to load all of them up from. Just tie it to a method that calls our api with a search term.